addons/pos_restaurant/models/pos_order.py

```diff
@@ -159,9 +159,10 @@ def get_table_draft_orders(self, table_id):
         :type table_id: int.
         :returns: list -- list of dict representing the table orders
         """
-        table_orders = self.search_read(
-                domain=[('state', '=', 'draft'), ('table_id', '=', table_id)],
-                fields=self._get_fields_for_draft_order())
+        domain = [('state', '=', 'draft')]
```

Comment:
this is bad because you're going to retrieve orders from **all** pos

Reply:
Thanks for your feedback.
I fixed it.